server/src/database/setup.ts

```diff
@@ -0,0 +1,70 @@
+import { readFile } from "fs";
+import parentLogger from "../../utils/logger";
+import db from "./db";
+import { CourseCode, PreReq, CoReq, CourseSection } from "./schema";
+
+const log = parentLogger.child({ module: "router" });
+const src = "./utils/output.json";
+
+export const setupDb = async () => {
+    await dropDb();
```

Comment:
We'll probably have to figure out a way to not reset the database every time someone starts the server :D.

Reply:
😂 Yeah haha wondering when/how we would be running these methods.

Reply:
I guess the other thing is I don't completely understand how this is going to work when we deploy (eg. updating the database when it's already deployed). I need to read up a bit more on this. I think for now this is okay. Maybe setupDB can check if there if the database is already populated and if that's the case, we dont do anything. I think dropDb only comes into play when we want to make an update to the db (eg. rescrape courses).